app/models/exam_authorization_request.rb

```diff
@@ -0,0 +1,24 @@
+class ExamAuthorizationRequest < ApplicationRecord
+  belongs_to :exam
+  belongs_to :user
+  belongs_to :organization
+
+  enum status: %i[pending approved rejected]
```

Comment:
Just nitpicking here, we usually use %i() notation instead of %i[]

Reply:
Ok, I didn't even know about the `%i[...]` syntax, Rubocop suggested it. Perhaps in the future you can include a Rubocop config file to enforce Mumuki style.